tests/server/routes/submitAssessmentRoute.test.js

```diff
@@ -0,0 +1,12 @@
+const request = require('supertest')
```

Comment:
Please change the name of this file to match the name of the file you are testing.
e.g tests/server/routes/assessments.test.js

Reply:
Done